### ISSUE TITLE
combination I + J Noto Sans

### ISSUE THREAD

**Issue by donderstorm** [googlei18n/noto-fonts#1113]
### Title 
  > The latin combination I + J is looking odd

### Font 
  > NotoSans (all versions)

### Font Version
  > * Mac - 1.04

### Issue
  > The combination I + J (capitals) which is used in Dutch frequently is looking very odd because the I has serifs and the J doesn't. 

### Character data
  > IJ 

### Screenshot
![screen shot 2018-03-27 at 16 50 39](https://user-images.githubusercontent.com/37841211/37975181-0443c1ce-31df-11e8-8e36-9eae9a273992.png)

**Comment by davelab6** [comment]
Maybe related to https://github.com/googlei18n/noto-fonts/issues/1002

**Comment by nizarsq** [comment]
This issue still reproducible on NotoSans and NotoSansMono 
<img width="486" alt="Screen Shot 2020-07-13 at 11 40 03 PM" src="https://user-images.githubusercontent.com/22158882/87392891-3121ff80-c562-11ea-9795-a97343f7a858.png">

**Comment by raoulunger** [comment]
I completely agree. The IJ combination is a separate letter in Dutch (on old typewriters the combination even had its own key). The issue is very noticeable as our main free newspaper (nu.nl) uses Noto Sans as its principle font. Have a look for example at this screenshot of today about the reliability of outdoor ice for skating ;-) :

![Screenshot 2021-02-12 at 16 08 08](https://user-images.githubusercontent.com/53795112/107785299-b36ae000-6d4c-11eb-95a2-7e11eb914af6.png)

There are many words in Dutch that start with IJ. We write the letter at the beginning of sentences by capitalizing both letters. That's where Noto Sans doesn't work well. The lower case ij has no problems.

Nu.nl is not overall a great example of good design choices (nor of a strong typographic conscience), but it makes the hiatus in the otherwise excellent Noto Sans quite visible. Would be great if the glyph could be added!
